analyzers/tests/SonarAnalyzer.UnitTest/Rules/InvocationResolvesToOverrideWithParamsTest.cs

```diff
@@ -28,10 +28,29 @@ public class InvocationResolvesToOverrideWithParamsTest
         private readonly VerifierBuilder builder = new VerifierBuilder<InvocationResolvesToOverrideWithParams>();
 
         [TestMethod]
-        public void InvocationResolvesToOverrideWithParams() =>
+        public void InvocationResolvesToOverrideWithParams()
+        {
+            var anotherAssembly = TestHelper.CompileCS("""
+                public class FromAnotherAssembly
+                {
+                    protected int ProtectedOverload(object a, string b) => 1493;
```

Comment:
Do these values carry any significance in the rule? I think they don't.

Let's remove the question "what 1493 means?" by using clear dummy value everywhere
```suggestion
                    protected int ProtectedOverload(object a, string b) => 42;
```

Reply:
This one wasn't addressed, it's in the test class (not a test case)

Reply:
Whoops, should all be fixed now.